src/gluonts/torch/model/deepar/lightning_module.py

```diff
@@ -47,13 +49,15 @@ def __init__(
         loss: DistributionLoss = NegativeLogLikelihood(),
         lr: float = 1e-3,
         weight_decay: float = 1e-8,
+        patience: Optional[int] = None,
```

Comment:
Maybe you can set the same good default as in the estimator here?

Reply:
👍